shared/cfg/guest-os/Linux/RHEL.cfg

```diff
@@ -20,3 +20,5 @@
             boot_path = ppc/ppc64
         x86_64, i386:
             kernel_params += " console=ttyS0,115200 console=tty0"
+        s390x:
+            kernel_params += " console=ttysclp0 debug ignore_loglevel"
```

Comment:
Is `debug` and `ignore_loglevel` meant to be permanent here? Either way, it'd be nice to explain the *why* for both `debug` and `ignore_loglevel` here.

Reply:
The same story as arm. As it's considered experimental (within Avocado-vt) I enabled debug in order to get more details during install. I can add the comment if you insist, but to me that looks unnecessary (there is no explanation of other params on other archs...)